docs/en_US/Compression/pruning.rst

```diff
@@ -0,0 +1,26 @@
+#################
+Pruning
+#################
+
+A common technique to increase sparsity in neural network model weights and activations is pruning.
+The pruning methods explore the redundancy in the model weights(parameters) and try to remove/prune the redundant and uncritical weights.
+The redundant elements are pruned from the model, their values are zeroed and we make sure they don't take part in the back-propagation process.
+
+From pruning granularity perspective, fine-grained pruning refers to pruning each individual weights seperately. 
+Coase-grained pruning or structured pruning is pruning entire group of elements, such as convolutional filter.
+
+NNI provides multiple fine-grained pruning and structured pruning algorithms.
```

Comment:
suggest aligning the terminology, such as fine-grained and coarse-grained pruning vs. unstructured pruning and structured pruning.

Reply:
Thanks, fixed.